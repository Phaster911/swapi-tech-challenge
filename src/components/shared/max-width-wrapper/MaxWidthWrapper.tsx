import styled from "styled-components";

const MaxWidthWrapper = styled.div`
  width: clamp(16rem, 95vw, 90rem);
`;

export default MaxWidthWrapper;
