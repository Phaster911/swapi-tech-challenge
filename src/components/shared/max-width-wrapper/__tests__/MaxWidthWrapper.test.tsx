import { render } from "@testing-library/react";
import MaxWidthWrapper from "../MaxWidthWrapper";

describe("<MaxWidthWrapper />", () => {
  it("renders the component", () => {
    const { container } = render(<MaxWidthWrapper />);
    expect(container).toMatchSnapshot();
  });
});
