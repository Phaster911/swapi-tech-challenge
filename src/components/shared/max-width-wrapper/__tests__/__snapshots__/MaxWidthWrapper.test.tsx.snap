// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<MaxWidthWrapper /> > renders the component 1`] = `
.c0 {
  width: clamp(16rem,95vw,90rem);
}

<div>
  <div
    class="c0"
  />
</div>
`;
